charms/worker/k8s/pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[project]
+name = "k8s"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
❤️

Reply:
i wonder if this is no longer needed
https://github.com/canonical/k8s-operator/pull/271